plenum/test/node_request/test_different_ledger_request_interleave.py

```diff
@@ -1,73 +1,111 @@
+                                             sdk_one_node_added,
+                                             tdir,
+                                             tdirWithPoolTxns,
+                                             allPluginsPath,
+                                             sdk_pool_handle, sdk_wallet_client,
+                                             sdk_wallet_steward):
     """
     Send pool and domain ledger requests such that they interleave, and do
     view change in between and verify the pool is functional
     """
-    new_node = one_node_added
-    sendReqsToNodesAndVerifySuffReplies(looper, wallet1, client1, 2)
+    new_node = sdk_one_node_added
+    sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle,
+                              sdk_wallet_client, 2)
     ensure_all_nodes_have_same_data(looper, txnPoolNodeSet)
 
     # Send domain ledger requests but don't wait for replies
-    requests = sendRandomRequests(wallet1, client1, 2)
+    requests = sdk_send_random_requests(looper, sdk_pool_handle, sdk_wallet_client, 2)
+
     # Add another node by sending pool ledger request
-    _, _, new_theta = nodeThetaAdded(looper, txnPoolNodeSet,
-                                     tdir, client_tdir,
-                                     tconf, steward1, stewardWallet,
-                                     allPluginsPath, name='new_theta')
+    _, new_theta = sdk_node_theta_added(looper,
+                                        txnPoolNodeSet,
+                                        tdir,
+                                        tconf,
+                                        sdk_pool_handle,
+                                        sdk_wallet_steward,
+                                        allPluginsPath,
+                                        name='new_theta')
 
     # Send more domain ledger requests but don't wait for replies
-    requests.extend(sendRandomRequests(wallet1, client1, 3))
+    requests.extend(sdk_send_random_requests(looper, sdk_pool_handle,
+                                             sdk_wallet_client, 3))
 
     # Do view change without waiting for replies
     ensure_view_change(looper, nodes=txnPoolNodeSet)
     checkProtocolInstanceSetup(looper, txnPoolNodeSet, retryWait=1)
 
     # Make sure all requests are completed
-    waitForSufficientRepliesForRequests(looper, client1,
-                                        requests=requests)
-
-    ensure_pool_functional(looper, txnPoolNodeSet, wallet1, client1)
-
-    new_steward, new_steward_wallet = addNewSteward(looper, client_tdir,
-                                                    steward1, stewardWallet,
-                                                    'another_ste')
+    total_timeout = sdk_eval_timeout(len(requests), len(txnPoolNodeSet))
+    sdk_replies = sdk_get_replies(looper, requests, timeout=total_timeout)
+    for req_res in sdk_replies:
+        sdk_check_reply(req_res)
```

Comment:
As far as I remember we have a special func to check bunch of replies. Try to use such func here instead of sdk_get_replies and for cycle